File: SoloLearn/Cheer Creator/CheerCreator.java
import java.util.Scanner;

public class CheerCreator {
    
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int yards = input.nextInt();
        if (yards > 10)
            System.out.println("High Five");
        else if (yards < 1)
            System.out.println("shh");
        else
            System.out.println("Ra!".repeat(yards));
        input.close();
    }
}

File: SoloLearn/Cheer Creator/CheerCreator.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoloLearn
{
    class Program
    {
        static void Main(string[] args)
        {
            int yards = Convert.ToInt32(Console.ReadLine());
            if (yards > 10) 
                Console.WriteLine("High Five");
            else if (yards < 1)
                Console.WriteLine("shh");
            else
            /*
            {
                string ra = "Ra!";
                Console.WriteLine(new StringBuilder(ra.Length * yards).Insert(0, ra, yards).ToString());
            }*/
                Console.WriteLine(String.Concat(Enumerable.Repeat("Ra!", yards)));
        }
    }
}